website/content/docs/release-notes/v0_12_0.mdx

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+page_title: v0.12.0
+description: |-
+  Boundary release notes for v0.12.0
+---
+
+# Boundary v0.12.0
+
+The release notes below contain information about new functionality available in the Boundary v0.12.0 release.
+To see a granular record of when each item was merged into the Boundary project, please refer to the [Changelog](https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md).
+To learn about what Boundary consists of, we highly recommend you review the [Getting Started Page](/boundary/docs/getting-started).
+
+For instructions on how to upgrade an existing Boundary deployment to v0.12.0, refer to Boundary's [general upgrade guide](/boundary/tutorials/oss-configuration/upgrade-version).
+
+## Boundary v0.12.0 highlights
+
+The 0.12.0 release adds major new functionality to Boundary.
+Highlights include:
+
+- Addresses on targets
+- Authentication improvements
+
+## New features
+
+**Addresses on targets**: HashiCorp Boundary offers an extensible domain model that allows administrators to organize target resources in a way that best compliments how their organization manages its computing infrastructure.
+However, that flexibility could sometimes become a hindrance when it comes to setting up a quick proof-of-concept, and defining a target to create a session.
```

Comment:
I don't want to argue over comma placement too much in a PR review, but I'm going to mark a few suggestions for comma removal that you can ignore if you'd like. 

I find that commas are often overused or used incorrectly in most technical writing, and make the text harder to read. Let's have a discussion about this offline sometime 💟

```suggestion
But that flexibility could become a hindrance when setting up a quick proof-of-concept and defining a target to create a session.
```